app/geo/gui/src/ts/twxplore/gui/geo/states/map/MapFeatureAttributeState/MapNumericFeatureAttributeState.ts

```diff
@@ -9,11 +9,11 @@ Unsure, if I should call this FeatureTypeAttributeState instead.
 Or maybe 'AttributeOfFeatureTypeState'
 */
 export interface MapNumericFeatureAttributeState {
-  range: MapFeatureAttribtueNumericRange;
+  range: MapFeatureAttributeNumericRange;
   filterIndex: number | null;
 }
 
-interface MapFeatureAttribtueNumericRange {
+interface MapFeatureAttributeNumericRange {
   min: number | null;
```

Comment:
This is incorrect. The semantics are that both min and max must be set or neither must be set. So what you really want is 

MapFeatureAttributeNumericRange { min: number; max: number; }
then above
range: MapFeatureAttributeNumericRange | null